idb/grpc/client.py

```diff
@@ -789,6 +791,48 @@ async def list_xctests(self) -> List[InstalledTestInfo]:
     async def send_events(self, events: Iterable[HIDEvent]) -> None:
         await self.hid(iterator_to_async_iterator(events))
 
+    @log_and_handle_exceptions
+    async def axtap(
+        self,
+        element: HIDElementType,
+        label: str,
+        x: float,
+        y: float,
+        duration: Optional[float] = None,
+        count: Optional[int] = 1,
+    ) -> None:
+        accessibilityInfo = await self.stub.accessibility_info(
+            AccessibilityInfoRequest(
+                point=None,
+                format=(AccessibilityInfoRequest.LEGACY),
+            )
+        )
+
+        elementFound = False
+
+        for item in json.loads(accessibilityInfo.json):
+            try:
+                axElement = HIDElementType(item["role_description"])
+                axLabel = item["AXLabel"]
+
+                elementFound = axElement == element and (
+                    label is None or axLabel == label
+                )
+
+                if elementFound:
+                    axframe = item["frame"]
+                    x += axframe["x"]
+                    y += axframe["y"]
+                    break
+            except:
```

Comment:
What could make it `raise`?

Reply:
I think the enum will `raise` a `ValueError` if it doesn't exist for the role:
`HIDElementType(item["role_description"])`

I'm happy to change that if there is a preferred way of handling this scenario.